### PR TITLE
Saving Messages

### PR DESCRIPTION
**Pull request by caiovictormc**
#### Updates
- Create settings.ini.example.
- Create `messages_tw.Message`.
- Update `BaseMessageSender` to save and manage outgoing messages.
- Create / Test MessageManager. repeatedly_recently to avoid flooding.
- Setup pytest.

### CODE REVIEW

File: messages_tw/views.py [hunk 1]
```diff
@@ -17,45 +20,77 @@ class BaseMessageSender(APIView):
...
+                body=message.content,
+            )
+        except TwilioException as e:
+            message.has_error = True
+            message.save()
+            raise e
+
+    def _send_message(self, to: str, message_text: str, kind: str):
+        if kind == Message.WHATSAPP:
+            message = Message(
```
Comment (victorvf) [inline messages_tw/views.py]: Aqui não precisa ser criado o Objeto também, igual ao else (Message.objects.create) ?

**Review by victorvf — APPROVED** [review]
LGTM
- just made a comment on the views

File: messages_tw/views.py (hunk 1 above)
Comment (caiovictormc) [reply messages_tw/views.py]: Yeap! Fixed!

**Comment by caiovictormc** [comment]
Your observation was correct, @victorvf ! Thanks! We need to create automated tests to cover our views. Can you perform this task?

**Pull request merged**